File: AdminAccount.java
public class AdminAccount {
    protected String username;
    private final String accessLevel = "admin";

    public AdminAccount(String username) {
        this.username = username;
    }

    public String displayName() {
        return username.trim().toUpperCase();
    }

    public boolean canManageUsers() {
        return accessLevel.equals("admin");
    }
}

// testing again! and again and again


// and again and againn

File: cpp/AdminAccount.cpp
#pragma once

#include <string>
#include "Account.cpp"

class AdminAccount : public Account {
private:
    std::string accessLevel = "admin";

public:
    explicit AdminAccount(std::string username)
        : Account(std::move(username)) {}

    bool canManageUsers() const {
        return accessLevel == "admin";
    }
};
